src/EntityFramework.SqlServer/SqlProviderServices.cs

```diff
@@ -115,23 +117,35 @@ public static SqlProviderServices Instance
         public static string SqlServerTypesAssemblyName { get; set; }
 
         /// <summary>
-        /// Set this flag to false to prevent <see cref="decimal" /> values from being truncated to
-        /// the scale (number of decimal places) defined for the column. The default value is true,
-        /// indicating that decimal values will be truncated, in order to prevent breaking existing
-        /// applications that depend on this behavior.
-        /// </summary>
-        /// <remarks>
-        /// With this flag set to true <see cref="SqlParameter" /> objects are created with their Scale
-        /// properties set. When this flag is set to false then the Scale properties are not set, meaning
-        /// that the truncation behavior of SqlParameter is avoided.
-        /// </remarks>
+        /// Get or sets a value indicating whether <see cref="decimal" /> parameter values are truncated to 
```

Comment:
Change this too to align with the structure recommended by @rowanmiller. 

cc @ajcvickers